protocol_ids.csv

```diff
@@ -4,6 +4,10 @@ Prefix,DisplayName,Authors,BitcoinAddress,SpecificationUrl,TxidRedirectUrl
 0x00584350,Counterparty Cash,Counterparty Cash Association (CCA),17YhxKgKunLjvi7HB1ENGmeLwPFiaxb74V,https://counterparty.cash,
 0x00746C6B,Keyport,Keyport,,https://keyport.io,
 0x02446365,SatoshiDICE,Jon Bestman,1DiceuELb5GZktc3CMEv868DMtU3B5957x,https://satoshidice.com,
+0x64617461,data,Steve Shadders,,,
+0x66696c65,file,Ryan X Charles,,,
```

Comment:
thanks for the credit :) minor issue, but for consistency it's best to have a period after the X - "Ryan X. Charles" :)